setup/setup.wxs

```diff
@@ -1,204 +1,204 @@
+                </CreateFolder>
+              </Component>
+
+              <Component Id="PKSimDB" Guid="686F86BE-F0C7-4F00-8982-4B376997D8BA">
+                <File Name="PKSimDB.sqlite" Id="PKSimDB" Source="$(var.DeployDir)/PKSimDB.sqlite" KeyPath="yes" />
+              </Component>
+
+              <Component Id='OSPSuite.Dimensions' Guid='79ED18B0-4CBC-45D3-A24F-16FC2B324409'>
+                <File Name='OSPSuite.Dimensions.xml' Id='OSPSuite.Dimensions' Source='$(var.DeployDir)/OSPSuite.Dimensions.xml' KeyPath='yes' />
+              </Component>
+
+              <Component Id='OSPSuite.PKParameters' Guid='1FB33C95-C00E-4C5C-B8C3-0D5E3C7B5034'>
+                <File Name='OSPSuite.PKParameters.xml' Id='OSPSuite.PKParameters' Source='$(var.DeployDir)/OSPSuite.PKParameters.xml' KeyPath='yes' />
+              </Component>
+
+              <Component Id="PKSimTemplateDBSystem" Guid="DF39FF9D-DAB8-4A6A-9E3E-FEFD68489F3C">
+                <File Name="PKSimTemplateDBSystem.templateDbSystem" Id="PKSimTemplateDBSystem" Source="$(var.DeployDir)/PKSimTemplateDBSystem.templateDbSystem" KeyPath="yes" />
+              </Component>
+
+              <Directory Id="ChartLayoutsFolder" Name="ChartLayouts">
+                <Component Id="ChartLayoutsFolder" Guid="FE800FCC-CF4A-452E-AA9D-E5DF4AF61777">
+                  <CreateFolder Directory="ChartLayoutsFolder">
+                    <Permission User="Everyone" GenericAll="yes" />
+                  </CreateFolder>
+                </Component>
+              </Directory>
+
+              <Directory Id="TeXTemplatesFolder" Name="TeXTemplates">
+                <Component Id="TeXTemplatesFolder" Guid="1EB74549-916C-42E1-B40B-199B43DF2ECC">
+                  <CreateFolder Directory="TeXTemplatesFolder">
+                    <Permission User="Everyone" GenericAll="yes" />
+                  </CreateFolder>
+                </Component>
+              </Directory>
+
+              <Directory Id="ExamplesFolder" Name="Examples">
+                <Component Id="ExamplesFolder" Guid="4CCE3113-8250-40AF-A9ED-7A9AC0CFED1B">
+                  <CreateFolder Directory="ExamplesFolder">
+                    <Permission User="Everyone" GenericAll="yes" />
+                  </CreateFolder>
+                </Component>
+
+                <Component Id="ExampleFolderShortCut" Guid="21CC8E07-4C3B-4613-B237-28F17160E474" Win64="yes">
+                  <Shortcut Id="ExampleFolderShortCut" Name="$(var.ProductName) Examples" Directory="OSPSuiteMenuDir" />
+                  <RegistryValue Root="HKCU" Key="Software\[Manufacturer]\[ProductName]" Type="string" Value="Examples" KeyPath="yes" />
+                </Component>
+              </Directory>
+            </Directory>
+          </Directory>
+        </Directory>
+      </Directory>
+
+      <!--Start menu Programms -->
+      <Directory Id="ProgramMenuFolder">
+        <Directory Id="OSPSuiteMenuDir" Name="$(var.SuiteName) $(var.SuiteVersion)">

```

Comment:
Also in the PRogram menu folder. Only one entry (9 or 10)
In the actual install folder, it is still 9.1 etc..